app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -414,7 +415,9 @@
     angular.element(pagination).css('display', 'none');
 
     if (viewType === 'grid' || viewType === 'tile') {
-      angular.element(mainContent).addClass('miq-sand-paper');
+      if (typeof formDiv === undefined || formDiv === null) {
```

Comment:
nitpick, but this could be just `if (! formDiv)`

Reply:
@h-kataria

Reply:
I was actually fixing that here recently: https://github.com/ManageIQ/manageiq-ui-classic/pull/5295/files#diff-a17816a0b00971e5816f379247a17f3dL99

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

says: 

> The typeof operator returns a string indicating ....